core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -83,6 +83,11 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val EVENT_LOG_PERMISSION =
+    ConfigBuilder("spark.eventLog.permission")
+      .stringConf
+      .createWithDefault("770")
```

Comment:
I think spark event log is always used for history server.There will be more frequent read by user in the same group than write,so I suggest the default permission is 750.

Reply:
But I don't understand this pr.Even the permissions of these event logs is 770, if some user want to read or write or the owner forbid other user read or write,the owner could change the permissions of these event logs by hadoop 'fs' cmd.This change looks like not required.

Reply:
I didn't change the default value for spark_event_log permission, the static config for this is 770.
Using the hadoop fs cmd can indeed solve the permission problem, but it also means the user needs to set the permissions every time they run Spark. It is awful